iOS/rainbow.xcodeproj/project.pbxproj

```diff
@@ -664,7 +661,6 @@
 				5305A1AC20A219380040A9A9 /* Logger.swift in Sources */,
 				53C6D18B20ADBB10008DF112 /* ProjectRainbowModel_1753554316.mlmodel in Sources */,
 				5305A1BA20A219380040A9A9 /* PersistableExtension.swift in Sources */,
-				A2B2CB2820AE117000A6F585 /* UIImageExtension.swift in Sources */,
```

Comment:
Just to be sure - we saw that this was a deleted class. We don't actually need this, right?

Reply:
we don't need that class